src/test/java/com/google/api/codegen/testdata/nodejs_test_library.baseline

```diff
@@ -565,6 +566,106 @@ describe('LibraryServiceClient', function() {
     });
   });
 
+  describe('streamShelves', function() {
+    it('invokes streamShelves without error', function(done) {
+      var client = libraryV1.libraryServiceClient();
+      // Mock request
+      var request = {};
+
+      // Mock response
+      var expectedResponse = {};
+
+      // Mock Grpc layer
+      var mockStream = new MockStream(request, expectedResponse);
+      client._streamShelves = function(actualRequest) {
+        assert.deepStrictEqual(actualRequest, request);
+        return mockStream;
+      };
+
+      var stream = client.streamShelves(request).on('data', function(response) {
+        assert.deepStrictEqual(response, expectedResponse);
+        done()
+      }).on('error', function(err) {
+        done(err);
+      });
+
+      stream.emitImmediately();
+    });
+  });
+
+  describe('streamBooks', function() {
+    it('invokes streamBooks without error', function(done) {
+      var client = libraryV1.libraryServiceClient();
+      // Mock request
+      var name = 'name3373707';
+      var request = {
+          name : name
+      };
+
+      // Mock response
+      var name2 = 'name2-1052831874';
+      var author = 'author-1406328437';
+      var title = 'title110371416';
+      var read = true;
+      var expectedResponse = {
+          name : name2,
+          author : author,
+          title : title,
+          read : read
+      };
+
+      // Mock Grpc layer
+      var mockStream = new MockStream(request, expectedResponse);
+      client._streamBooks = function(actualRequest) {
+        assert.deepStrictEqual(actualRequest, request);
+        return mockStream;
+      };
+
+      var stream = client.streamBooks(request).on('data', function(response) {
+        assert.deepStrictEqual(response, expectedResponse);
+        done()
+      }).on('error', function(err) {
+        done(err);
+      });
+
+      stream.emitImmediately();
+    });
+  });
+
+  describe('discussBook', function() {
+    it('invokes discussBook without error', function(done) {
+      var client = libraryV1.libraryServiceClient();
+      // Mock request
+      var name = 'name3373707';
+      var request = {
+          name : name
+      };
+
+      // Mock response
+      var userName = 'userName339340927';
+      var comment = '95';
+      var expectedResponse = {
+          userName : userName,
+          comment : comment
+      };
+
+      // Mock Grpc layer
+      var mockStream = new MockStream(request, expectedResponse);
```

Comment:
`mockStream` does not need to be declared for BIDI streaming since the 'done' event will be triggered on stream.write.